app/controllers/bridges_controller.rb

```diff
@@ -1,4 +1,49 @@
 # frozen_string_literal: true
 
 class BridgesController < ApplicationController
+  before_action :authorize_request
+  before_action :set_bridge, only: %i[show update destroy]
+
+  def index
+    render_message message: @current_user.bridges.all
+  end
+
+  def show
+    render_message message: @bridge.to_json(include: %i[headers environment_variables events])
+  end
+
+  def create
+    @bridge = Bridge.new(bridge_params)
+    @bridge.user = @current_user
+
+    if @bridge.save
+      render_message status: :created
+    else
+      render_message message: @bridge.errors, status: :internal_server_error
```

Comment:
This isn't an internal server error, this is a bad request

Reply:
You should never throw 500s. Rails will throw a 500 for you when you fail to catch an error.